ECommerceAPI/Data/Models/AttributeValue.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ECommerceAPI.Data.Models
+{
+    public class AttributeValue
```

Comment:
I am not sure this required in this style

Reply:
We talked about this previously, so that when a new product is added, for example its colors or size ..etc are added

Reply:
ok so lets assume the model 
{
id,
Name,
Value}

Reply:
done